examples/random_forest_example.py

```diff
@@ -0,0 +1,169 @@
+import shapely
+
+from openeo.processes import ProcessBuilder, mean, array_create
+from shapely.geometry import Polygon
+import openeo
+from openeo.udf import FeatureCollection
+from geopandas import GeoDataFrame
+
+polygon1 = Polygon([(16.138916, 48.1386), (16.138916, 48.320647), (16.2, 48.320647), (16.2, 48.1386)])
+polygon2 = Polygon([(16.3, 48.1386), (16.3, 48.320647), (16.5, 48.320647), (16.5, 48.1386)])
+polygons = shapely.geometry.GeometryCollection([polygon1, polygon2])
```

Comment:
GeoJSON GeometryCollection should be avoided in most (if not all, practically) cases

see https://github.com/Open-EO/openeo-processes/pull/253

for a single "entity" consisting of multiple polygons, use MultiPoligon
for multiple entities, use FeatureCollection